pyspectral/utils.py

```diff
@@ -284,13 +286,13 @@ def download_rsr(dest_dir=None, dry_run=False):
     functions from the internet as tarballs, extracts them, then deletes
     the tarball.
 
-    See :func:`pyspectral.rsr_reader.check_and_download` for a "smart" version
+    See: func: `pyspectral.rsr_reader.check_and_download` for a "smart" version
     of this process that only downloads the necessary files.
 
     Args:
-        dest_dir (str): Path to put the temporary tarball and extracted RSR
+        dest_dir(str): Path to put the temporary tarball and extracted RSR
```

Comment:
Also not sure if this was automated or not, but I'm pretty sure it is required/standard that there be a space between the argument name and data type.

Reply:
Really weird! That must have been automated. And be due to the pre-commit stuff.